src/parse/asp/objects.go

```diff
@@ -37,7 +38,11 @@ type freezable interface {
 // Not all pyObjects implement this.
 type iterable interface {
 	pyObject
-	// This isn't super generic but it works fine for all cases we have right now.
+	Iter() iter.Seq[pyObject]
+}
+
+// An indexable represents an object that knows its length and can be indexed into.
+type indexable interface {
```

Comment:
Potentially missing `pyObject` here?

Reply:
It doesn't really matter as such - most of the others don't have it. I'll make them consistent.